### DISCUSSION TITLE
Stanford CS224w: Machine Learning with Graphs - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#174] [Announcements]
# Stanford CS224w: Machine Learning with Graphs - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%B7%B1%E5%BA%A6%E5%AD%A6%E4%B9%A0/CS224w/

**Comment by zhaochenyang20** [comment]
老师年轻帅气，英语好似来自南亚

**Reply by whuLames** [reply]
俄罗斯人 一股毛子味英语

**Reply by MTDickens** [reply]
好像是斯洛文尼亚？不过都是东欧口音

**Comment by MisterPANDC** [comment]
有课程的notes吗？找了好久没找到

**Reply by agjycxysow** [reply]
https://snap-stanford.github.io/cs224w-notes/
official notes here.  :)

**Comment by usamimeri** [comment]
现在点进链接是2023版本的，这个版本没有对应的视频，只有slide和colab，建议使用2021版本的课程搭配视频使用，以2023课程的slide作为补充资料
https://web.stanford.edu/class/cs224w/

**Comment by JackBo04** [comment]
笑死我了，油管翻译插件先把老师的口音识别为印地语了
